.github/workflows/depcheck.yml

```diff
@@ -0,0 +1,27 @@
+name: Depcheck
+
+on:
+  push:
+    branches:
+      - nextjs
+  pull_request:
+    branches:
+      - nextjs
+
+jobs:
+  check_dependencies:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: 18
```

Comment:
sweep: don't we need to install yarn after getting npm to even be able to use it?